cli/azd/cmd/init.go

```diff
@@ -178,8 +178,8 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 	followUp := heredoc.Docf(`
 	You can view the template code in your directory: %s
 	Learn more about running 3rd party code on our DevHub: %s`,
-		output.WithLinkFormat("%s", wd),
-		output.WithLinkFormat("%s", "https://aka.ms/azd-third-party-code-notice"))
+		output.WithHyperlink(wd, ""),
+		output.WithHyperlink("https://aka.ms/azd-third-party-code-notice", ""))
```

Comment:
Can we go further and remove all urls?  
Like, in this case, we would use the `3rd party code` text to become a link, and remove the extra text?

Reply:
We could.  I left the `aka.ms` links in there since they are typically short enough to display without wrapping.  
I defer to @savannahostrowski & @Austinauth here.

Reply:
@vhvb1989 I'd prefer if we hold off on removing all URLs until we test with a screen reader user and/or have a verbose fallback to force printing the URL.